src/OVAL/public/oval_session.h

```diff
@@ -194,15 +193,14 @@ OSCAP_API int oval_session_evaluate_id(struct oval_session *session, char *probe
  *
  * @memberof oval_session
  * @param session an \ref oval_session
- * @param probe_root FIXME:
  * @param fn a callback function
  * @param arg an optional argument for your callback function
  *
  * @retval 0 on success
  * @retval 1 on an internal error (use \ref oscap_err_desc or \ref
  * oscap_err_get_full_error to get more details)
  */
-OSCAP_API int oval_session_evaluate(struct oval_session *session, char *probe_root, agent_reporter fn, void *arg);
+OSCAP_API int oval_session_evaluate(struct oval_session *session, agent_reporter fn, void *arg);
```

Comment:
I think we could also remove  `void *arg` because it is NULL at the only place where it is called.

Reply:
IMO that's scope creep, it's used for the agent_reporter so let's keep it at least in this PR.

Reply:
OK